pages/agent/v3/ssh_keys.md.erb

```diff
@@ -6,7 +6,7 @@ If your agent needs to clone your repositories using git and SSH, you'll need to
 
 ## Finding your SSH key directory
 
-When the Buildkite Agent runs any git operations, SSH will look for keys in `~/.ssh` under whichever user the agent runs as. Each platform's [agent installation documentation](/docs/agent/v3/installation) specifies the exact location of this directory.
+The Buildkite agent runs [on your own machine](/docs/agent/v3/installation), whether it's a VPS, server, desktop computer, embedded device. When the Buildkite agent runs any git operations, SSH will look for keys in `~/.ssh` under whichever user the agent runs as. Each platform's [agent installation documentation](/docs/agent/v3/installation) specifies the exact location of this directory.
```

Comment:
I think this is actually less helpful here, even though it's a direct copy of the content in `installation.md.erb`. How about something more like 

> When the Buildkite agent runs any git operations, it looks for SSH keys in `~/.ssh` under the user the agent runs as. Each platform's [agent installation documentation](/docs/agent/v3/installation) specifies which user the agent runs as and which directory the SSH keys are in. For example, on Debian the agent runs as `buildkite-agent` and the SSH keys are in `/var/lib/buildkite-agent/.ssh/` but on macOS the agent runs as the user who started the `lauchd` service, and the SSH keys are in that users `.ssh` directory

Do we think the example is helpful here?

Reply:
Implemented. Not sure about adding an extra example.